### DISCUSSION TITLE
Changing env directory since Symfony 5.3

### DISCUSSION THREAD

**Discussion by DocFX** [symfony/symfony#42444] [Q&A] [answered]
Hello,

We've always managed to change the .env source directory for all Symfony projects, and the `index.php` file has totally changed in 5.3. 

How are we supposed to achieve that now?

Thanks!

**Comment by faizanakram99** [comment, marked as answer]
You can set `dotenv_path` to set location of `.env` files. It can be set either in `$_SERVER['APP_RUNTIME_OPTIONS']` or in `composer.json` file (in `extra.runtime` section)

```php
$_SERVER['APP_RUNTIME_OPTIONS'] = [
    'dotenv_path' => '/path/to/dotenv/files`
]
```

See https://symfony.com/doc/current/components/runtime.html#using-options for details.

**Reply by DocFX** [reply]
Hahaha I JUST found out at the same time! :)

Thanks a lot!

Also: don't forget to do it in `console` file as well :)

**Reply by faizanakram99** [reply]
If you set it in `composer.json` file, you don't need to modify `index.php` or `console` file. Useful if the path is static.

**Reply by DocFX** [reply]
Wow, this can be overloaded in the `composer.json`? I had the idea, but didn't think further. 

How would you achieve this? Through `extra` configuration?

**Reply by faizanakram99** [reply]
yep, I did mention it in answer :smiley: 

```json
  "extra": {
    "runtime": {
      "dotenv_path": "/path/to/dot/env"
    }
},
```
